services/motion/builtin/builtin_test.go

```diff
@@ -235,6 +235,65 @@ func TestMoveSingleComponent(t *testing.T) {
 	})
 }
 
+func TestMoveOnMapLongDistance(t *testing.T) {
+	ctx := context.Background()
+	logger := golog.NewTestLogger(t)
+	injectSlam := inject.NewSLAMService("test_slam")
+
+	const chunkSizeBytes = 1 * 1024 * 1024
+
+	injectSlam.GetPointCloudMapFunc = func(ctx context.Context) (func() ([]byte, error), error) {
+		path := filepath.Clean(artifact.MustPath("slam/example_cartographer_outputs/viam-office-02-22-3/pointcloud/pointcloud_4.pcd"))
```

Comment:
Do we have a helper for this chunk yet?  If not, would be good to make

Reply:
oh yes that got merged in with my other PR, will change in this test too